examples/logging/tutorial-server-prometheus/README.md

```diff
@@ -0,0 +1,157 @@
+
+## 2. Setup Prometheus/Grafana Stack
+
+For simplicity, we have provided `docker-compose.yaml` that spins up the containerized Prometheus/Grafana stack. In that file, we instruct `prometheus.yaml` (a [Prometheus config file](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)) to be passed to the Prometheus container. Inside `prometheus.yaml`, the `scrape_config` has the information about the `metrics` endpoint exposed by the server on port `6100`.
+
+<details>
+    <summary>Click to see Docker Compose file looks like</summary>
+
+```yaml    
+# docker-compose.yaml
+    
+version: "3"
+
+services:
+  prometheus:
+    image: prom/prometheus
+    extra_hosts:
+      - "host.docker.internal:host-gateway"     # allow a direct connection from container to the local machine
+    ports:
+      - "9090:9090" # the default port used by Prometheus
+    volumes:
+      - ${PWD}/prometheus.yaml:/etc/prometheus/prometheus.yml # mount Prometheus config file
+
+  grafana:
+    image: grafana/grafana:latest
+    depends_on:
+      - prometheus
+    ports:
+      - "3000:3000" # the default port used by Grafana
+
+```
+</details>
+
+<details>
+    <summary>Click to see Prometheus Config file looks like</summary>
+    
+```yaml
+# prometheus.yaml
+    
+global:
+  scrape_interval: 15s                      # how often to scrape from endpoint
+  evaluation_interval: 30s                  # time between each evaluation of Prometheus' alerting rules
+
+scrape_configs:
+  - job_name: prometheus_logs               # your project name
+    static_configs:
+      - targets:
+          - 'host.docker.internal:6100'     # should match the port exposed by the PrometheusLogger in the DeepSparse Server config file 
+```
+</details>
+
+To start up a Prometheus stack to monitor the DeepSparse Server, run:
+
+```bash
+cd docker
+docker-compose up
+```
+
+## 3. Launch the Python Client and Run Inference
+
+`client.py` is a simple client, that periodically sends requests to the Server. The client simulates the behavior of some application, 
+that sends the raw inputs to the inference server and receives the outputs. 
+
+Run the following to start inference:
+
+```bash
+python client/client.py client/piglet.jpg 5543
+```
+The first argument is the path to the sample image, while the second argument is the port number that matches the inference endpoint of the Server.
+
+## 4. Inspecting the Prometheus/Grafana Stack
+
+You may visit `localhost:9090` to inspect whether Prometheus recognizes the `metrics` endpoint (`Status` -> `Targets`)
+
+![img.png](images/img_1.png)
+
+Visit `localhost:3000` to launch Grafana. Log in with the default username (`admin`) and password (`admin`). 
+Setup the Prometheus data source (`Add your first data source` -> `Prometheus`). 
+
+Now you should be ready to create/import your dashboard. If you decide to import the dashboard, either upload `grafana/dashboard.json` or 
+paste its contents using Grafana's `import` functionality.
```

Comment:
Let's test whether this is going to work for the user off-the bat. I might have some dashboard settings cached when I used import, so this did not reflect the "clean slate" setup that the user will be coming with.